initAuth.js

```diff
@@ -0,0 +1,43 @@
+import { init } from "next-firebase-auth";
+
+const initAuth = () => {
+  init({
+    authPageURL: "/",
+    appPageURL: "/auth",
+    loginAPIEndpoint: "/api/login", // required
+    logoutAPIEndpoint: "/api/logout", // required
+    // firebaseAuthEmulatorHost: "localhost:9099",
+    // Required in most cases.
+    firebaseAdminInitConfig: {
+      credential: {
+        projectId: process.env.FIREBASE_PROJECT_ID,
```

Comment:
you will need to add a .env.local file relative to root directory for this to work properly.  i will send a .env.local file in teams chat for you all to just paste into your root directory.  The reason I added it an .env is because these are private keys and must be kept private